views/models/ParticipantsModel/index.js

```diff
@@ -0,0 +1,140 @@
+const ParticipantsModel = {
+  selectDropdownOptions: (participants) =>
+    participants.map(({ study, subject }) => ({
+      value: `${subject}`,
+      label: `${subject} in ${study}`,
+    })),
+  sortAsc: function (currentParticipant, nextParticipant, sortBy) {
```

Comment:
How many participants is a given user going to have to deal with? Ideally I think we would do sorting on the back end as part of the Mongo query. This would be faster and allow us to add pagination. However if the current implementation does everything on the front end I would consider that a new feature rather than an issue for this PR.

Reply:
The way it is currently written the user will deal with all of the subjects for the sites they have access to. 
if site A has 20 
and site B has 200
Joe has access to just site A he will see 20
Mac has access to site A and B he will see 220